app/src/lib/stores/app-store.ts

```diff
@@ -1985,16 +1990,28 @@ export class AppStore {
         gitHubRepository.endpoint
       )
       if (account) {
+        const hasValidToken =
+          account.token.length > 0 ? 'has token' : 'empty token'
```

Comment:
I'm doing this because I'm pretty sure some of our authentication issues are due to the account and stored token being out of sync (we use the credential store, and rehydrate it when retrieving accounts at launch), but lack the data to confirm it.

Reply:
Also, we totally could just be returning an empty token with `GIT_ASKPASS`:

https://github.com/desktop/desktop/blob/5a638b712a0c88758a5e3b6c04860fa1565884b6/app/src/ask-pass/ask-pass.ts#L21-L22